test/README.md

```diff
@@ -47,3 +46,41 @@ Teardown:
 ```
 $ rake vagrant:teardown
 ```
+
+## Vagrant test environment using Docker Compose
+
+This environment is built from sources with CoreOS running in Virtualbox.
+
+Setup:
+
+```
+test $ vagrant up
+test $ vagrant ssh
+core@localhost $ cd /kontena/test
+core@localhost /kontena/test $ docker-compose run --rm test rake compose:setup
+```
+
+Specs:
+
+```
+core@localhost /kontena/test $ docker-compose run --rm test rake
+```
+
+or
+
+```
+core@localhost /kontena/test $ docker-compose run --rm test rspec spec/
+```
+
+Teardown:
+
+```
+core@localhost /kontena/test $ docker-compose run --rm test rake compose:teardown
+test $ vagrant destroy
```

Comment:
Yes, this is for development, when running e2e specs against some PR branch, or writing new e2e specs. The `test/Vagrantfile` is to provide a CoreOS system to run those local e2e specs with local modifications, without me having to run weave directly on my Ubuntu machine...